### ISSUE TITLE
vertical_parameter_alignment generates a warning in nested function calls

### ISSUE THREAD

**Issue by Pispa339** [realm/SwiftLint#1818]
Version 0.22.0

Aligning parameters in nested function calls with an additional indent generates a vertical_parameter_alignment warning, although one could argue that being the more readable format.

example:
```
static var someStaticVar: SomeObject {
            return SomeObject(
                logo: logo,
                title: NSLocalizedString("Localized string", comment: "comment"),
                subtitle: NSLocalizedString(
                    "A longer localized string of which's parameters wouldn't fit on one line.",
                     comment: "A comment on the longer localized string of which's parameters wouldn't fit on one line."),
                someLabel "some label")
        }
```
generates a warning

**Comment by marcelofabri** [comment]
The problem isn't about nested functions, it's just that `comment` is one space after "A longer...".
If you update to this snippet, no violations are triggered:

```swift
static var someStaticVar: SomeObject {
    return SomeObject(
        logo: logo,
        title: NSLocalizedString("Localized string", comment: "comment"),
        subtitle: NSLocalizedString(
            "A longer localized string of which's parameters wouldn't fit on one line.",
            comment: "A comment on the longer localized string of which's parameters wouldn't fit on one line."),
        someLabel "some label")
}
```

**Comment by Pispa339** [comment]
Sorry, that was just a bad paste/formatting in my issue description. I'm getting the warning with the updated snippet as well.

**Comment by marcelofabri** [comment]
Are you sure? Can you post a full snippet that triggers the issue?

**Comment by Pispa339** [comment]
Double checked everything once more. Seems like a case of PEBCAK. My apologies.
